### PR TITLE
Missing logging added 

### PR DESCRIPTION
**Pull request by iasatan**

### CODE REVIEW

File: service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java [hunk 1]
```diff
@@ -91,23 +95,18 @@ public HttpRequestExecutingMessageHandler httpGateway() {
         gateway.setHttpMethod(HttpMethod.GET);
         gateway.setExpectedResponseType(String.class);
         gateway.setOutputChannel(setAlgorithmReplyChannel());
+        LOG.info("algorithm set in Positioning server using Spring Integration");
```
Comment (judit0310) [inline service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java]: Please start this sentence with capital letter.

File: service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java [hunk 2]
```diff
@@ -91,23 +95,18 @@ public HttpRequestExecutingMessageHandler httpGateway() {
...
         return gateway;
     }
 
     @Bean
     @ServiceActivator(inputChannel = "getLocationQueryChannel")
     public HttpRequestExecutingMessageHandler locationGateway() {
-        //System.out.println("fsaz");
```
Comment (judit0310) [inline service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java]: I don't know why this command had been committed in the first place.

File: controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java [hunk 3]
```diff
@@ -45,6 +48,7 @@ public MobileTransferDTO mobileSendMeasurementHandler(@RequestBody(required = fa
 		
 		UserSecurityDetails userDetails = (UserSecurityDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
 		System.out.println(userDetails.toString());
+		LOG.info("trackPosition in MobileRequestsController. user:"+userDetails.toString());
```
Comment (judit0310) [inline controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java]: Please start it with capital letter.

File: service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java (hunk 2 above)
Comment (iasatan) [reply service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java]: oops

File: controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java [hunk 4]
```diff
@@ -45,6 +48,7 @@ public MobileTransferDTO mobileSendMeasurementHandler(@RequestBody(required = fa
 		
 		UserSecurityDetails userDetails = (UserSecurityDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
 		System.out.println(userDetails.toString());
```
Comment (judit0310) [inline controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java]: The userDetails is why printed out? What is the purpose of this command?

File: service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/ZoneQueryServiceSIConfig.java [hunk 5]
```diff
@@ -120,6 +124,7 @@ public HttpRequestExecutingMessageHandler listGateway() {
         gateway.setHttpMethod(HttpMethod.GET);
         gateway.setExpectedResponseType(Collection.class);
         gateway.setOutputChannel(listZonesReplyChannel());
+        LOG.info("list Zones query was requested from measurement server with Spring Integration");
```
Comment (judit0310) [inline service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/ZoneQueryServiceSIConfig.java]: Please start the sentence with capital letter.

File: service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/ZoneQueryServiceSIConfig.java [hunk 6]
```diff
@@ -103,11 +107,11 @@ public HttpRequestExecutingMessageHandler httpGateway() {
         Map<String, Expression> uriVariableExpressions = new HashMap<>(1);
         uriVariableExpressions.put("zoneID", expressionParser.parseExpression("headers['zoneID']"));
         HttpRequestExecutingMessageHandler gateway = new HttpRequestExecutingMessageHandler("http://" + System.getProperty("measurement.host") + ":" + System.getProperty("measurement.port") + "/zones/{zoneID}");
-//       HttpRequestExecutingMessageHandler gateway = new HttpRequestExecutingMessageHandler("http://localhost:8081/zones/183f0204-5029-4b33-a128-404ba5c68fa8");
         gateway.setUriVariableExpressions(uriVariableExpressions);
         gateway.setHttpMethod(HttpMethod.GET);
         gateway.setExpectedResponseType(ZoneDTO.class);
         gateway.setOutputChannel(getZoneReplyChannel());
+        LOG.info("get zone by id query was requested from measurement server with Spring Integration");
```
Comment (judit0310) [inline service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/ZoneQueryServiceSIConfig.java]: Please start it with capital letter.

File: controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java (hunk 4 above)
Comment (iasatan) [reply controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java]: I don't know, I did't wrote it

**Pull request merged**